src/stylesheets/elements/_tags.scss

```diff
@@ -13,7 +14,6 @@
 }
 
 .usa-tag-big {
-  font-size: $base-font-size;
-  padding-left: 0.9rem;
-  padding-right: 0.9rem;
+  @include u-padding-x(1);
+  @include u-font('sans', 'xs');
```

Comment:
Annnd, this makes me want to set up our default to use relative sizes for vars like `$theme-base-font-size` — right now it's assigned `6` but it could/should be `xs`...

Reply:
Done in fe3ca1e. But also it wasn't clear we could do this from the documentation. Or would it be better if it was: `@include u-font-size('body', 'sm');` (I cant tell what body size should be bc it says body is Merriweather in the docs, when it should be SSP)?